autoscheduler/frontend/src/components/LandingPage/HelpText/HelpText.tsx

```diff
@@ -1,25 +1,29 @@
 import * as React from 'react';
 import LargeTextCard from '../../LargeTextCard/LargeTextCard';
 
-const bodyText = `
-Somebody once told me the world is gonna roll me
-I ain't the sharpest tool in the shed
-She was looking kind of dumb with her finger and her thumb
-In the shape of an "L" on her forehead
-Well the years start coming and they don't stop coming
-Fed to the rules and I hit the ground running
-Somebody once told me the world is gonna roll me
-I ain't the sharpest tool in the shed
-She was looking kind of dumb with her finger and her thumb
-In the shape of an "L" on her forehead
-Well the years start coming and they don't stop coming
-Fed to the rules and I hit the ground running`;
-
-const HelpText: React.SFC = () => (
+const HelpText: React.FC = () => (
   <LargeTextCard
-    title="Guidelines"
+    title="Welcome to Rev Registration!"
   >
-    {bodyText}
+    <div style={{
+      gap: 50,
+    }}
+    >
+      <p>
+        Simply select a term, pick your courses, mark off when you&apos;re not available, we&apos;ll
```

Comment:
```suggestion
        Rev Registration is an automatic schedule generator for Texas A&M. Simply select a term, pick your courses, mark off when you&apos;re not available, we&apos;ll
```

Reply:
Didn't we agree on `automatic class scheduler`?

Reply:
Yeah I'm pretty confident we did

Reply:
oops, my bad, I wrote down the wrong version. `automatic class scheduler` is perfect, no changes needed